Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -187,7 +187,18 @@ private void CreateMapVote(IPlayerSession? initiator)
                         Loc.GetString("ui-vote-map-win", ("winner", maps[picked])));
                 }
 
-                _gameMapManager.TrySelectMap(picked.ID);
+                var ticker = EntitySystem.Get<GameTicker>();
```

Comment:
EntitySystem.Get<T> is deprecated.

Reply:
`GameTicker` does not exist until the round starts so binding it when `VoteManager` starts causes an exception. Can that be delayed somehow?

Reply:
resolved